sql/rpl_record.cc

```diff
@@ -400,6 +400,36 @@ size_t pack_row(TABLE *table, MY_BITMAP const *columns_in_image,
   DBUG_RETURN(static_cast<size_t>(pack_ptr - row_data));
 }
 
+static void insert_row_fields(std::string *row_query, TABLE *table) {
```

Comment:
There is no checking for `row_query != nullptr`.
Pass inout parameters by reference
```suggestion
static void insert_row_fields(std::string &row_query, TABLE *table) {
```